### ISSUE TITLE
Feature Request: Find reference among modified code

### ISSUE THREAD

**Issue by Rhynorater** [differentmatt/vscode-copy-github-url#26]
When you have added additional lines to the code base, the lines numbers get thrown off. It would be an awesome feature if the application tried to identify the desired line within the last committed version of the file from the repo. If this isn't already a feature, I'd be happy to put a bounty on it.

Thanks,
Justin

**Comment by differentmatt** [comment]
No, this isn't a feature yet.  Good idea though!  Marking as enhancement for future work.

Thanks for the input.
